src/pages/junghyowon/Login/JungLogin.js

```diff
@@ -1,7 +1,103 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import './JungLogin.scss';
 
 function JungLogin() {
-  return <div>정효원 로그인페이지</div>;
+  const [id, setId] = useState('');
+  const [password, setPw] = useState('');
+  const navigate = useNavigate();
+  const idPw = id.indexOf('@') !== -1 && password.length > '5';
+
+  const onCheckEnter = e => {
+    if (e.key === 'Enter') {
+      goToMain();
+    }
+  };
+
+  const goToMain = e => {
+    navigate('/JungMain');
+  };
+
+  const idValue = e => {
+    setId(e.target.value);
+  };
+
+  const pwValue = e => {
+    setPw(e.target.value);
+  };
+
+  const loginFetch = () => {
+    fetch('http://10.58.52.248:3001/auth/signup', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json;charset=utf-8',
+      },
+      body: JSON.stringify({ email: id, password: password }),
+    })
+      .then(response => console.log(response))
+      .then(data => localStorage.setItem('hyowon', data));
+  };
+
+  return (
+    <>
+      <section className="container-wrap">
+        <h1>Westagram</h1>
+        <form className="input-container">
+          <div className="input-wrap">
+            {/*로그인 아이디 */}
+            <input
+              id="id"
+              type="text"
+              placeholder="전화번호, 사용자이름 또는 이메일"
+              onChange={idValue}
+            />
+            {/*패스워드 */}
+            <input
+              id="password"
+              type="password"
+              placeholder="비밀번호"
+              onChange={pwValue}
+              onKeyUp={onCheckEnter}
+            />
```

Comment:
```suggestion
            <input
              type="text"
              placeholder="전화번호, 사용자이름 또는 이메일"
              onChange={idValue}
              name="id"
              value={userInfo.id}
            />
            <input
              name="password"
              value={userInfo.password}
              type="password"
              placeholder="비밀번호"
              onChange={pwValue}
            />
```
- 2개의 상태를 하나로 관리하고 하나의 함수로 진행하게 된다면 input에도 속성을 넣어줘야 합니다.
- onKeyUp은 없어도 될 것 같습니다.
- id는 고유해야합니다. 리액트 특성상 컴포넌트 재활용 될 가능성이 많기 때문에 특정한 상황이 아니라면 id대신 className 사용을 추천합니다.